src/anemoi/training/config/dataloader/native_grid.yaml

```diff
@@ -45,6 +45,8 @@ training:
   frequency: ${data.frequency}
   drop:  []
 
+validation_rollout: 1 # number of rollouts to use for validation, must be equal or greater than rollout expected by callbacks
```

Comment:
Can this number be lower than the training rollout without overwriting the training rollout steps?